### PR TITLE
fix svg viewBox bug 🎱

### PR DESCRIPTION
**Pull request by mesqueeb**
The main problem was here:
image

This should be set to the canvas width and height that the dev has chosen.

### CODE REVIEW

**Comment by mesqueeb** [comment]
fixes #439

File: src/signature_pad.ts [hunk 1]
```diff
@@ -563,9 +563,7 @@ export default class SignaturePad {
       '<svg' +
       ' xmlns="http://www.w3.org/2000/svg"' +
       ' xmlns:xlink="http://www.w3.org/1999/xlink"' +
-      ` viewBox="${minX} ${minY} ${maxX} ${maxY}"` +
-      ` width="${maxX}"` +
-      ` height="${maxY}"` +
```
Comment (jmeinke) [inline src/signature_pad.ts]: @mesqueeb Why did you remove width and height?

File: .gitignore [hunk 2]
```diff
@@ -1,5 +1,4 @@
 node_modules/
-dist/
```
Comment (jmeinke) [inline .gitignore]: @mesqueeb Why are you adding the dist folder?

File: src/signature_pad.ts (hunk 1 above)
Comment (mesqueeb) [reply src/signature_pad.ts]: Sorry!
I never include it on my SVGs, viewBox should be enough.

File: .gitignore (hunk 2 above)
Comment (mesqueeb) [reply .gitignore]: I can't remember! 😅

**Comment by mesqueeb** [comment]
@jmeinke I have decided to close my crappy PR lol.
Please review and merge this PR instead. It was trying to solve the same issue as me:

https://github.com/szimek/signature_pad/pull/411

**Comment by jmeinke** [comment]
Unfortunately, I don't have the permissions to merge it.
